src/App.tsx

```diff
@@ -1,25 +1,23 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { TodoListItem } from "./TodoListItem";
+
+const todos: Todo[] = [
```

Comment:
great start 🎉 👍 

Next steps would be moving those todos to state (`useState`) and adding input for creating new todos :)

Reply:
UseState now in action, sir. Input for new actions will be available soon. Please be patient.